js/ui/appDisplay.js

```diff
@@ -626,6 +633,10 @@ var AllView = class AllView extends BaseAppView {
     getLayoutIds() {
         let layoutIds = super.getLayoutIds();
 
+        // Only show the hack icon if the clubhouse app is in the system
+        if (_getClubhouse())
+            layoutIds.splice(0, 0, HACK_APP_ID);
```

Comment:
This could cause side effects. If an element is not found an empty array is returned. No items on the desktop? :package: 
I think we need to guard this.

Reply:
At least when installing/uninstalling the Clubhouse I have seen traces for this in the log and empty desktop grids.

Reply:
I think that the problem was not there. I've been reviewing this and I've detected that the `hackAppIcon` variable was not set to null on redraw, causing a crash. I've fixed that and tested installing and uninstalling the clubhouse and now it seems to work correctly.

Reply:
Great!